src/Components/Buttons/Default.tsx

```diff
@@ -76,8 +79,7 @@ export const StyledButton = styled(Button)`
     }};
   }
 
-  ${fonts.primary.style}
-  ${block()}
+  ${fonts.primary.style} ${block()};
```

Comment:
Minor, but adding a semicolon after fonts to prevent Prettier changes might be nicer to read